coding-standards/UI.md

```diff
@@ -0,0 +1,505 @@
+         <td align="left" style="height: 139px;"> </td>
+         <td align="left" style="height: 139px;">Left</td>
+         <td align="left" style="height: 139px;">Numbers with slash(/) or Hyphen(-)or dot(.).</td>
+         <td align="left" style="height: 139px;">Depends on the customer's requirement.  
+            Must be less than the current date.
+         </td>
+      </tr>
+      <tr style="height: 157px;">
+         <td align="left" style="height: 157px;">StartDate</td>
+         <td align="left" style="height: 157px;">Datetime</td>
+         <td align="left" style="height: 157px;"> </td>
+         <td align="right" style="height: 157px;">10</td>
+         <td align="left" style="height: 157px;"> </td>
+         <td align="left" style="height: 157px;">Left</td>
+         <td align="left" style="height: 157px;">Numbers with slash(/) or Hyphen(-)or dot(.).</td>
+         <td align="left" style="height: 157px;">Depends on the customer's requirement.  
+            Should be greater than the current date.
+         </td>
+      </tr>
+      <tr style="height: 157px;">
+         <td align="left" style="height: 157px;">EndDate</td>
+         <td align="left" style="height: 157px;">Datetime</td>
+         <td align="left" style="height: 157px;"> </td>
+         <td align="right" style="height: 157px;">10</td>
+         <td align="left" style="height: 157px;"> </td>
+         <td align="left" style="height: 157px;">Left</td>
+         <td align="left" style="height: 157px;">Numbers with slash(/) or Hyphen(-)or dot(.).</td>
+         <td align="left" style="height: 157px;">Depends on the customer's requirement.  
+            Must be greater than or equals to startdate
+         </td>
+      </tr>
+      <tr style="height: 49px;">
+         <td align="left" style="height: 49px;">Error Label</td>
+         <td align="left" style="height: 49px;"> </td>
+         <td align="left" style="height: 49px;"> </td>
+         <td align="left" style="height: 49px;"> </td>
+         <td align="left" style="height: 49px;"> </td>
+         <td align="left" style="height: 49px;"> </td>
+         <td align="left" style="height: 49px;"> </td>
+         <td align="left" style="height: 49px;">Must be in <span style="color: #ff0000;">RED</span> color.</td>
+      </tr>
+      <tr style="height: 49px;">
+         <td align="left" style="height: 49px;">Message Label</td>
+         <td align="left" style="height: 49px;"> </td>
+         <td align="left" style="height: 49px;"> </td>
+         <td align="left" style="height: 49px;"> </td>
+         <td align="left" style="height: 49px;"> </td>
+         <td align="left" style="height: 49px;"> </td>
+         <td align="left" style="height: 49px;"> </td>
+         <td align="left" style="height: 49px;">Must be in <span style="color: #008000;">GREEN</span> color.</td>
+      </tr>
+   </tbody>
+</table>
+
+# Standard regular expressions to be used for form validations across the technologies
```

Comment:
If we want to keep this here then delete the regex file which we created before: 

https://github.com/rajkumar-osmosys/dev-standards/blob/patch-1/coding-standards/RegEx.md